### PR TITLE
[NFC][CIFuzz] Fix coding issues.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Fix some coding and style issues.
1. Treat "testcase" as one word. This is consistent with ClusterFuzz.
2. Treat "stacktrace" as one word. This is consistent with ClusterFuzz.
3. Fix spelling of comments and words in code.
4. Use descriptive style docstrings and remove unnecessary clutter
(e.g. "Test the functionality of foo_function function in bar module"
becomes "Tests foo_function."
5. Use JSON for a list needed in testing instead of pickle.
6. Use `file_handle` as the name of a variable representing a file handle instead of `file`.
7. Fix wrong capitalization.
8. Share more code in tests.
9. Fix some bad integration tests that were broken by changes to the outside world.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
I think the failures are unrelated to my changes. Will fix in another PR.

**Comment by jonathanmetzman** [comment]
Ping

File: infra/build_specified_commit_test.py [hunk 1]
```diff
@@ -72,8 +72,15 @@ def test_build_fuzzers_from_commit(self):
 
   def test_detect_main_repo_from_commit(self):
     """Test the detect main repo function from build specific commit module."""
-    for example_repo in test_repos.TEST_REPOS:
+    # TODO(metzman): Fix these tests so they don't randomly break because of
+    # changes in the outside world.
+    test_repos_list = [
+        repo for repo in test_repos.TEST_REPOS if repo.project_name != 'usrsctp'
```
Comment (inferno-chromium) [inline infra/build_specified_commit_test.py:78]: Can you add a comment why usrsctp is explicitly excluded.

File: infra/cifuzz/actions/build_fuzzers/build_fuzzers_entrypoint.py [hunk 2]
```diff
@@ -17,8 +17,7 @@
 import os
 import sys
 
-# pylint: disable=wrong-import-position
-# pylint: disable=import-error
+# pylint: disable=wrong-import-position,import-error
```
Comment (inferno-chromium) [inline infra/cifuzz/actions/build_fuzzers/build_fuzzers_entrypoint.py:20]: is this a simple fix, i see this same pylint in a bunch of files.

**Review by inferno-chromium — APPROVED** [review]
Nice cleanup!

**Pull request merged**